server/spec/lib/data_import/my_anime_list/extractor/character_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.describe DataImport::MyAnimeList::Extractor::Character do
+  let(:guts_character) { fixture('my_anime_list/characters/guts-character.html') }
+  let(:faye_valentine_character) { fixture('my_anime_list/characters/faye-valentine-character.html') }
+  let(:kyou_aguri_character) { fixture('my_anime_list/characters/kyou-aguri-character.html') }
+  let(:sachi_nanjou_character) { fixture('my_anime_list/characters/sachi-nanjou-character.html') }
```

Comment:
These names don't need to be so verbose, `faye_valentine_character` could probably reasonably be `faye`